templates/js-langchain/src/main.js

```diff
@@ -0,0 +1,86 @@
+import { Actor } from 'apify';
+import { ApifyDatasetLoader } from "langchain/document_loaders/web/apify_dataset";
+import { Document } from "langchain/document";
+import { HNSWLib } from "langchain/vectorstores/hnswlib";
+import { OpenAIEmbeddings } from "langchain/embeddings/openai";
+import { RetrievalQAChain } from "langchain/chains";
+import { OpenAI } from "langchain/llms/openai";
+
+import { retrieveVectorIndex, cacheVectorIndex } from './vector_index_cache.js';
+
+await Actor.init();
+
+const {
+    startUrls = [{ url: 'https://wikipedia.com' }],
+    maxCrawlPages = 3,
+    forceRecrawl = false, // Enforce a re-crawl of website content and re-creation of the vector index.
+    query = 'What is Wikipedia?',
+} = await Actor.getInput() || {};
+
+// There are 2 environment variables you need to configure (https://docs.apify.com/cli/docs/vars#set-up-environment-variables-in-apify-console):
+// 1. Apify API token you obtain at https://console.apify.com/account/integrations. This is not nessesary when running this Actor at Apify platform.
```

Comment:
It's called `APIFY_TOKEN`, not `APIFY_API_TOKEN` on the platform.
https://github.com/apify/apify-shared-js/blob/master/packages/consts/src/consts.ts#L264

Reply:
Honestly we should just provide both `APIFY_TOKEN` and `APIFY_API_TOKEN` on the platform, I make this mistake a lot 😄